gatsby/src/pages/404.tsx

```diff
@@ -1,12 +1,26 @@
 import React from "react";
 import { Layout } from "../components/Layout/Layout";
 import { SEO } from "../components/SEO/SEO";
+import { Link } from "gatsby";
 
 const NotFoundPage: React.FC = () => {
 	return (
 		<Layout>
 			<SEO title="Page not found" noIndex={true} />
-			<h1>Page not found</h1>
+			<h1>We can't find this page</h1>
+			<p>Check that the web address has been typed correctly.</p>
+			You can also try:
+			<ul>
+				<li>Look for it using the search box</li>
+				<li>
+					Browse for it from the <Link to="/">CKS homepage.</Link>
```

Comment:
Can you use lowercase b, change to browsing and moving the full stop outside the link
```suggestion
					browsing for it from the <Link to="/">CKS homepage</Link>.
```